core/storage/suggestion/gae_models.py

```diff
@@ -481,16 +482,46 @@ def get_in_review_translation_suggestions(
             language_codes: list(str). The list of language codes.
 
         Returns:
-            list(SuggestionModel). A list of suggestions that are of the given
-            type, which are in review, but not created by the given user.
+            list(SuggestionModel). A list of suggestions which are in review,
```

Comment:
Should probably say "translation suggestions".

Reply:
Done